contracts/src/messaging/messenger/CrossChainEnabledObscuro.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./ICrossChainMessenger.sol";
+
+// TODO: We need to upgrade the open zeppelin version to the 4.x that adds cross chain enabled
```

Comment:
is this todo done ?

Reply:
Not quite yet, the signature verification from attestators needs touching past 4.7.2 i think.